org.eclipse.xtext.ui.shared/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,7 @@ Bundle-Version: 2.13.0.qualifier
 Bundle-Localization: plugin
 Bundle-Activator: org.eclipse.xtext.ui.shared.internal.Activator
 Bundle-Vendor: Eclipse Xtext
-Require-Bundle: org.eclipse.xtext.ui;resolution:=optional;x-installation:=greedy,
+Require-Bundle: org.eclipse.xtext.ui;bundle-version="[0.0.0,2.14.0)";resolution:=optional;x-installation:=greedy,

```

Comment:
if i have a look at the linked bugzilla it looks like we should require a minimal version as well (if we go this path) @szarnekow what do you think?